fog/ledger/server/src/bin/key_image_store.rs

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) 2018-2022 The MobileCoin Foundation
+
+use clap::Parser;
+use grpcio::{RpcStatus, RpcStatusCode};
+use mc_common::{logger::log, time::SystemTimeProvider};
+use mc_fog_ledger_enclave::{LedgerSgxEnclave, ENCLAVE_FILE};
+use mc_fog_ledger_server::{KeyImageStoreServer, LedgerStoreConfig};
+use mc_ledger_db::LedgerDB;
+use mc_util_grpc::AdminServer;
+use mc_watcher::watcher_db::WatcherDB;
+
+use std::{env, sync::Arc};
+
+fn main() {
+    let (logger, _global_logger_guard) =
+        mc_common::logger::create_app_logger(mc_common::logger::o!());
+    mc_common::setup_panic_handler();
+    let config = LedgerStoreConfig::parse();
+
+    let enclave_path = env::current_exe()
+        .expect("Could not get the path of our executable")
+        .with_file_name(ENCLAVE_FILE);
+    log::info!(
+        logger,
+        "enclave path {}, responder ID {}",
+        enclave_path
+            .to_str()
+            .expect("enclave path is not valid UTF-8"),
+        &config.client_responder_id
+    );
```

Comment:
Consider pulling out the two format values into variables and then use the variables inline in the format string.